### DISCUSSION TITLE
[Question][Hw1][Problem 5] There is a test case error on homework pdf

### DISCUSSION THREAD

**Discussion by Hhro** [prosyslab-classroom/cs524-program-analysis#407] [Q&A]
The third example on pdf seems wrong. The result should be 'TyError'.

<img width="598" alt="image" src="https://user-images.githubusercontent.com/36997987/132350678-b3b9a343-8106-4362-a1b3-afc372222d2c.png">

**Comment by KihongHeo** [comment]
You are right. That is a typo.

**Comment by hyunsukimsokcho** [comment]
Closing.. please feel free to re-open if needed.
